internal/commands/sbomcreate/depgraph.go

```diff
@@ -35,7 +35,7 @@ func GetDepGraph(ictx workflow.InvocationContext) (*DepGraphResult, error) {
 	}
 	depGraphs, err := engine.InvokeWithConfig(DepGraphWorkflowID, depGraphConfig)
 	if err != nil {
-		return nil, errFactory.NewDepGraphWorkflowError(err)
+		return nil, errFactory.NewDepGraphWorkflowGenerationError(err)
```

Comment:
I don't think this rename is necessary, because it is about an error coming back from the `DepGraph` workflow. I don't think this requires mentioning the "generation" part.

Reply:
I've added the generation part cause the error string mentions that it occurs when generating an SBOM, but I agree, this is a more generic error.